### ISSUE TITLE
Vanilla style UI

### ISSUE THREAD

**Issue by HowardZHY** [UnlegitMC/FDPClient#1949]
These fancy things caused most crashes and render bugs. The best way to prevent these crashes and render bugs is to remove fancy backgrounds and fonts at all, make it just as simple as Wurst 6.

**Comment by JonnyOnline** [comment]
Yes but you should bei able to activate fancy ui

**Comment by HowardZHY** [comment]
no, it always crashes when finishing launching and FDP folder didn't generate so i can't config anything

**Comment by HowardZHY** [comment]
now there are less render bugs/crashes on 3.1.1, so we can add a mode to hide all FDP things except rshift gui
(nobody knows u're using FDP until opened rshift gui)
